[Build] [WebGPU] Failed to build without "--enable_wasm_threads"

### Describe the issue

The latest code fails to build WebGPU without "--enable_wasm_threads". But if we add "--enable_wasm_threads" (Whole command becomes "build.bat --config Release --build_wasm --skip_tests --parallel --skip_submodule_sync --disable_wasm_exception_catching --use_jsep --use_webnn --target onnxruntime_webassembly --enable_wasm_simd --enable_wasm_threads"), the build is successful. 

### Urgency

This should impact the release.

### Target platform

Windows

### Build script

build.bat --config Release --build_wasm --skip_tests --parallel --skip_submodule_sync --disable_wasm_exception_catching --use_jsep --use_webnn --target onnxruntime_webassembly --enable_wasm_simd

### Error / output

FAILED: CMakeFiles/onnxruntime_mlas.dir/D_/workspace/project/tmp/onnxruntime/onnxruntime/core/mlas/lib/q4_dq.cpp.o
D:\workspace\project\tmp\onnxruntime\cmake\external\emsdk\upstream\emscripten\em++.bat -DBUILD_MLAS_NO_ONNXRUNTIME -DEIGEN_MPL2_ONLY -DEIGEN_USE_THREADS -DNSYNC_ATOMIC_CPP11 -DORT_ENABLE_STREAM -DPLATFORM_POSIX -DUSE_JSEP=1 -DUSE_WEBNN=1 -ID:/workspace/project/tmp/onnxruntime/build/Windows/Release/_deps/utf8_range-src -ID:/workspace/project/tmp/onnxruntime/include/onnxruntime -ID:/workspace/project/tmp/onnxruntime/include/onnxruntime/core/session -ID:/workspace/project/tmp/onnxruntime/build/Windows/Release/_deps/google_nsync-src/public -ID:/workspace/project/tmp/onnxruntime/build/Windows/Release -ID:/workspace/project/tmp/onnxruntime/onnxruntime -ID:/workspace/project/tmp/onnxruntime/build/Windows/Release/_deps/abseil_cpp-src -ID:/workspace/project/tmp/onnxruntime/onnxruntime/core/mlas/inc -ID:/workspace/project/tmp/onnxruntime/onnxruntime/core/mlas/lib -ID:/workspace/project/tmp/onnxruntime/build/Windows/Release/_deps/gsl-src/include -ffunction-sections -fdata-sections -flto -msimd128 -O3 -DNDEBUG -std=gnu++17 -fPIC -Wall -Wextra -Wno-unused-parameter -Wno-deprecated-copy -Wno-tautological-pointer-compare -Wno-ambiguous-reversed-operator -Wno-deprecated-anon-enum-enum-conversion -Wno-undefined-var-template -Wno-deprecated-builtins -Wshorten-64-to-32 -Werror -MD -MT CMakeFiles/onnxruntime_mlas.dir/D_/workspace/project/tmp/onnxruntime/onnxruntime/core/mlas/lib/q4_dq.cpp.o -MF CMakeFiles\onnxruntime_mlas.dir\D_\workspace\project\tmp\onnxruntime\onnxruntime\core\mlas\lib\q4_dq.cpp.o.d -o CMakeFiles/onnxruntime_mlas.dir/D_/workspace/project/tmp/onnxruntime/onnxruntime/core/mlas/lib/q4_dq.cpp.o -c D:/workspace/project/tmp/onnxruntime/onnxruntime/core/mlas/lib/q4_dq.cpp
D:/workspace/project/tmp/onnxruntime/onnxruntime/core/mlas/lib/q4_dq.cpp:751:9: error: use of undeclared identifier 'ORT_ENFORCE'
  751 |         ORT_ENFORCE(zero_points || signed_quant, "Unsigned quant with no zero points is not supported.");
      |         ^
D:/workspace/project/tmp/onnxruntime/onnxruntime/core/mlas/lib/q4_dq.cpp:798:9: error: use of undeclared identifier 'ORT_ENFORCE'
  798 |         ORT_ENFORCE(
      |         ^
D:/workspace/project/tmp/onnxruntime/onnxruntime/core/mlas/lib/q4_dq.cpp:832:9: error: use of undeclared identifier 'ORT_ENFORCE'
  832 |         ORT_ENFORCE(columns % 2 == 0, "Columns must be multiple of 2.");
      |         ^
D:/workspace/project/tmp/onnxruntime/onnxruntime/core/mlas/lib/q4_dq.cpp:846:17: error: use of undeclared identifier 'ORT_ENFORCE'
  846 |                 ORT_ENFORCE(buffer_size == thread_blk_size, "buffer size must be equal to thread block size.");
      |                 ^
D:/workspace/project/tmp/onnxruntime/onnxruntime/core/mlas/lib/q4_dq.cpp:1099:9: error: use of undeclared identifier 'ORT_ENFORCE'
 1099 |         ORT_ENFORCE(columns % 2 == 0, "Columns must be multiple of 2");
      |         ^
D:/workspace/project/tmp/onnxruntime/onnxruntime/core/mlas/lib/q4_dq.cpp:1760:9: error: use of undeclared identifier 'ORT_THROW'
 1760 |         ORT_THROW("Row-wise MlasQDQQuantizeBlockwise is not implemented");
      |         ^
D:/workspace/project/tmp/onnxruntime/onnxruntime/core/mlas/lib/q4_dq.cpp:1812:9: error: use of undeclared identifier 'ORT_THROW'
 1812 |         ORT_THROW("Row-wise MlasQDQTransposeBlockwiseQuantized is not implemented");
      |         ^
7 errors generated.

### Visual Studio Version

_No response_

### GCC / Compiler Version

_No response_

--- a/onnxruntime/core/mlas/lib/q4common.h
+++ b/onnxruntime/core/mlas/lib/q4common.h
@@ -18,6 +18,8 @@ Abstract:
     the parameters and the quantized data in a blob.
 --*/
 
+#include "core/common/common.h"
+
 #include "mlas_q4.h"
 #include "mlasi.h"
 

